src/agent/src/rpc.rs

```diff
@@ -1641,21 +1641,26 @@ fn setup_bundle(cid: &str, spec: &mut Spec) -> Result<PathBuf> {
     let config_path = bundle_path.join("config.json");
     let rootfs_path = bundle_path.join("rootfs");
 
-    fs::create_dir_all(&rootfs_path)?;
-    BareMount::new(
-        &spec_root.path,
-        rootfs_path.to_str().unwrap(),
-        "bind",
-        MsFlags::MS_BIND,
-        "",
-        &sl!(),
-    )
-    .mount()?;
+    let does_bundle_exist = Path::new(&rootfs_path).is_file();
+    info!(sl!(), "Does the bundle exist {:?}", does_bundle_exist);
+    info!(sl!(), "The rootfs_path is {:?}", rootfs_path);
+
+    if !does_bundle_exist {
```

Comment:
Could we still do the:

```Rust
if !Path::new(&rootfs_path).exists() {
}
```

modification?

Reply:
Sorry, I slightly misunderstood what you mean last time. Hopefully the change I've made now is ok?